manifest.json

```diff
@@ -3,60 +3,70 @@
     "theme": "activation",
     "new_version": "1.1.1",
     "url": "http://wpnux.godaddy.com/?theme=activation",
-    "package": "https://github.com/godaddy/wp-activation-theme/releases/download/v1.1.1/activation.zip"
+    "package": "https://github.com/godaddy/wp-activation-theme/releases/download/v1.1.1/activation.zip",
+    "screenshot": "https://raw.githubusercontent.com/godaddy/wp-activation-theme/master/screenshot.png"
```

Comment:
@EvanHerman The `screenshot` should reference the release tag, not `master`.

e.g. https://raw.githubusercontent.com/godaddy/wp-activation-theme/v1.1.1/screenshot.png